common/src/main/java/com/wynntils/models/rewards/RewardsModel.java

```diff
@@ -72,12 +72,23 @@ public ItemAnnotation fromCharmItemStack(ItemStack itemStack, StyledText name, S
     public TomeItem fromTomeItemStack(ItemStack itemStack, StyledText name) {
         GearTier gearTier = GearTier.fromStyledText(name);
 
-        TomeInfo tomeInfo = tomeInfoRegistry.getFromDisplayName(name.getStringWithoutFormatting());
+        String tomeName = name.getStringWithoutFormatting();
+        boolean isUnidentified = false;
+        if (tomeName.startsWith("Unidentified ")) {
+            tomeName = tomeName.substring("Unidentified ".length());
+            isUnidentified = true;
+        }
```

Comment:
Honestly the annotator could pass in the real item name. I really dislike doing string manipulations, they are so easily broken.

Reply:
Would this require `fromTomeItemStack` to additionally take in a boolean argument indicating whether the tome is unidentified?